woocommerce/woocommerce
# Remove insight on first sale note

### All Submissions:






### Changes proposed in this Pull Request:

Removes the insight note.

Part of #34930 .






### How to test the changes in this Pull Request:



1. Create a store between 1 and 4 weeks old.
1. Install a plugin like [WP Crontrol](https://wordpress.org/plugins/wp-crontrol/)
1. Under Tools -> Cron Events, run wc_admin_daily
1. Make sure the note with a title of "Did you know?" has not been added



### Other information:




### FOR PR REVIEWER ONLY:
